### ISSUE TITLE
Update to newest sfntly

### ISSUE THREAD

**Issue by madig** [daltonmaag/sfntly-py#2]
Recently hit some EOT conversion crash that I don't want to debug because using upstream git fixed it.

**Comment by madig** [comment]
Wait. The problem only goes away when I disable MTX compression for specific fonts. Oh well.

**Comment by anthrotype** [comment]
is https://github.com/googlei18n/sfntly/pull/108 related?

better to update anyway

Also... why are we still shipping EOT?

**Comment by madig** [comment]
Nope. Some experimentation of the font designer showed that the problem is actually missing/empty hinting tables. These trigger a NullPointerException in the glyf encoding code somewhere it seems.

A legacy never dies :)

**Comment by anthrotype** [comment]
oh, then please open an issue on google/sfntly with a reproducer if you have time
otherwise nevermind
